doc/proposals/openshift-4.3/operator-testing-tool.md

```diff
@@ -0,0 +1,240 @@
+---
+title: operator-testing-tool
+authors:
+  - "@jmccormick2001"
+  - "@joelanford"
+reviewers:
+  - TBD
+  - "@joelanford"
+approvers:
+  - TBD
+creation-date: 2019-09-13
+last-updated: 2019-09-13
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+see-also:
+  - "/enhancements/this-other-neat-thing.md"  
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# operator-testing-tool
```

Comment:
```suggestion
## New Operator Testing Tool for Operator SDK
```

Reply:
Also, after starting to read it will not help the users/devs impl testing. I understand that it is a tool to check its QA. So, WDYT about :

```suggestion
New Operator QA Tool for Operator SDK (Scorecard improvements)
```

Reply:
I would suggest that we match the epic name directly.
```suggestion
# Tooling for Testing Operators
```

Reply:
updated in the next version.